src/components/calcite-tile-select-group/calcite-tile-select-group.tsx

```diff
@@ -15,6 +28,30 @@ export class CalciteTileSelectGroup {
   /** Tiles by default move horizontally, stacking with each row, vertical allows single-column layouts */
   @Prop({ reflect: true }) layout?: TileSelectGroupLayout = "horizontal";
 
+  /**
+   * Fired when tiles are selected/deselected
+   * all checked tiles are passed in the event detail
+   */
+  @Event() calciteTileSelectGroupChange: EventEmitter<HTMLCalciteTileSelectElement[]>;
+
+  @Listen("calciteRadioButtonChange")
+  radioButtonChangeHandler(event: CustomEvent): void {
+    this.emitChange(event);
+  }
+
+  @Listen("calciteCheckboxChange")
+  checkboxChangeHandler(event: CustomEvent): void {
+    this.emitChange(event);
+  }
+
+  private emitChange(event: CustomEvent): void {
+    event.stopPropagation();
+    const selected = nodeListToArray(this.el.querySelectorAll("calcite-tile-select")).filter(
```

Comment:
Rather than use `querySelectorAll` here we should be able to setup the radio and checkbox custom events to emit the new values.  Perhaps that might be a more streamlined way of obtaining the new checked items?

Reply:
Nitpick: I think we could get rid of `nodeListToArray` and use `Array.from` by relying on the types.